malcolm/modules/ADCore/parts/positionlabellerpart.py

```diff
@@ -18,37 +18,42 @@
 if TYPE_CHECKING:
     from typing import Tuple
 
+# Pull re-used annotypes into our namespace in case we are subclassed
```

Comment:
Not needed as we don't override __init__

Reply:
Although this does violate the "all annotypes should be there if we are subclassed" principle, but it's probably clearer to only pull annotypes into our namespace if we are using them